ui/src/assets/docs/de/developer-portal-page.md

```diff
@@ -0,0 +1,364 @@
+# Developer Portal
+
+---
+
+## Einleitung
+
+Die [Payment Service Directive 2 (PSD2)](https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CELEX:32015L2366&from=EN)
+schreibt Banken einen einheitlichen Standard für eine Account-Schnittstelle erreichbar für Drittanbieter vor. Im weiteren Text wird diese Account-Schnittstelle als "XS2A" bezeichnet, was für Acces-to-Acount" steht. <!-- vielleicht wäre ein Glossar hilfreich? -->  
```

Comment:
Die [Payment Service Directive 2 (PSD2)](https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CELEX:32015L2366&from=EN) schreibt Banken vor eine standardisierte Account-Schnittstelle für Drittanbieter (TPPs) bereitzustellen.

Reply:
done

Reply:
it's missing?